addons/mail/static/src/model/sample_server.js

```diff
@@ -0,0 +1,13 @@
+/* @odoo-module */
+
+import { SampleServer } from "@web/model/sample_server";
+import { patch } from "@web/core/utils/patch";
+
+patch(SampleServer.prototype, {
+    _getRandomSelectionValue(modelName, field) {
```

Comment:
Would keep the small docstring explanation part that mentions we don't want to show a bunch of warnings in the list.